python/metashape_workflow_functions.py

```diff
@@ -931,7 +937,7 @@ def build_model(self):
                         fileh.write(
                             ", ".join(str(transform_tuple[i * 4 : (i + 1) * 4]))
                         )
-
+#export 
```

Comment:
nitpick: generally comments should be at the same level of indentation as the code. In this case it looks like it may be notes to yourself that could be removed entirely.